test/unittests/UserTest.php

```diff
@@ -0,0 +1,526 @@
+<?php
+/**
+ * Unit tests for User class
+ *
+ * PHP Version 5
+ *
+ * @category Tests
+ * @package  Main
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+require_once __DIR__ . '/../../vendor/autoload.php';
+use PHPUnit\Framework\TestCase;
+/**
+ * Unit test for User class
+ *
+ * @category Tests
+ * @package  Main
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class UserTest extends TestCase
+{
+    /**
+     * The username for the private user object
+     *
+     * @var string
+     */
+    private $_username;
+    /**
+     * Stores user information
+     *
+     * @var array
+     */
+    private $_userInfo
+        = array('ID'                     => 1,
+                'UserID'                 => '968775',
```

Comment:
It's not necessary but consider using constants for these values instead of "magic numbers". It'll make this code a little easier to maintain if the values change, for whatever reason.